### PR TITLE
[PATCH v3] travis: add out-of-tree builds

### PR DESCRIPTION
**Pull request by muvarov**
Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

### CODE REVIEW

**Comment by lumag** [comment]
/usr/bin/install: cannot stat ‘../../../doc/implementers-guide/implementers-guide.html’: No such file or directory
make[3]: *** [install-docDATA] Error 1

File: doc/implementers-guide/Makefile.am [hunk 1]
```diff
@@ -5,7 +5,7 @@ TARGET = implementers-guide.html
 
 EXTRA_DIST = $(SRC)
 
-doc_DATA = $(TARGET)
+doc_DATA = $(top_srcdir)/doc/implementers-guide/$(TARGET)
```
Comment (lumag) [inline doc/implementers-guide/Makefile.am]: I think this can be as simple as
```make
doc_DATA = $(srcdir)/$(TARGET)
```
Will this work?

Comment (muvarov) [inline doc/implementers-guide/Makefile.am]: @lumag no, it does not work https://travis-ci.org/muvarov/odp/jobs/247575518 also the same full patch is for other docs.

**Comment by lumag** [comment]
@muvarov Just a second thought. Overall pull request seems incorrect. OOT builds are done during `make distcheck` Travis target. We just need to expand `AM_DISTCHECK_OPTIONS`. I'll send replacement PR soon.

**Comment by muvarov** [comment]
#lumag ok than check that fix captures error fixed with first commit.

**Comment by lumag** [comment]
See #61

**Comment by lumag** [comment]
@muvarov I think this can be closed now.

**Comment by muvarov** [comment]
Closed.

**Pull request closed without merging**